### PR TITLE
December product growth strategy update

### PR DESCRIPTION
**Pull request by malomarrec**
This is based on discussions with the team, and learnings from the past month.
Will be merged on 2022-12-06 8PM PST.

### CODE REVIEW

File: content/strategy-goals/strategy/growth-team/index.md [hunk 1]
```diff
@@ -20,17 +22,27 @@ Our key metric is PQLs (Product Qualified Leads), as defined by (WIP):
...
+  - It's hard and high commitment for users to add their code. It's frequent for trials to be in fact _pre-trials_: a potential champion requests a trial, and adds their personal code to build a case to convince their team or management to run a Sourcegraph trial. We need to make that motion very easy, both in-product, and by allowing those champions to self-serve what they need (security, one-pager, business case, etc) to drive a Sourcegraph trial.
+  - There are many friction points in the flow, that are likely to cause churn. For example, if initial users don't reset their password within the first 72 hours, they need to talk to a CE to get a new reset password link.
+  - The signup workflow is very manual, and relies on lots of ad hoc automation powered by Zapier that is brittle and constrains the experience. Managing trials requires lots of time from CE.
+
+- **Virality**
+  - There's no easy/practical way to invite collaborators and users to a cloud instance. Besides, unless SSO is setup and `allowSignup` is true, users can't join a pre-existing trial instance. That means unless the initial admins is very proactive, others won't join the instance.
```
Comment (sqs) [inline content/strategy-goals/strategy/growth-team/index.md:39]: I think this flow would be awesome (and meet users' expectations):

- I sign in via Google Workspace or GitHub to start a trial on signup.sourcegraph.com
- If Google Workspace, my instance is preconfigured to allow other users to sign in via Google Workspace
- If GitHub, my instance is preconfigured to allow other users to sign in via GitHub

File: content/strategy-goals/strategy/growth-team/index.md [hunk 2]
```diff
@@ -39,16 +51,16 @@ Our overall strategy is to unlock the four key themes that will enable product-l
...
+- **Iterate on the signup experience.** Build the infrastructure required to improve the signup experience ([RFC 763](https://docs.google.com/document/d/1MoDYlWBWALGDj-DJdlCvAAFpkrW0nwJL_nb0ih4uUJ0/edit#)). We will start building a cloud console API that will progressively replace the Zapier backend, and both reduce the operational burden, and improve the experience (eg. the brittle initial reset password experience).
+- **Refine metrics**
 
 ### What we're not working on & why
 
-- We are not working on the self-hosted motion in September and October, as we are focused on cloud.
-- We are not building in-product conversion flows (example: credit card payments), and focus on generating PQLs for Sourcegraph's go-to-market team to engage with.
+- We are not working on the self-hosted motion until we have a working self-serve Cloud motion.
+- We are not building in-product conversion flows (example: credit card payments), and focus on generating PQLs for Sourcegraph's go-to-market team to engage with (product-led sales).
+- We're not immediately working on virality, until we fix initial onboarding. We need to focus on the biggest problem first.
```
Comment (erzhtor) [inline content/strategy-goals/strategy/growth-team/index.md:78]: 👍

**Review by erzhtor — APPROVED** [review]

File: content/strategy-goals/strategy/growth-team/index.md (hunk 1 above)
Comment (malomarrec) [reply content/strategy-goals/strategy/growth-team/index.md:39]: Great point. This is the endgame and should be clearer in the page, updating.
Achieving this type of flow is one of the reasons the infrastructure [here](https://github.com/sourcegraph/handbook/pull/5717/files#diff-fdfa5563fd5508f35f1bbc523a056e5ec10ef625b7c4d7079ccf9efa6e654605R59) is super important. We want to be able to configure instances such as this, or adding OSS code by default (currently blocked: https://github.com/sourcegraph/sourcegraph/pull/44398)

**Comment by malomarrec** [comment]
I intend to merge this by Tuesday 2022-12-06 8 PM PST, feedback welcome!

File: content/strategy-goals/strategy/growth-team/index.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
+In a year, Sourcegraph will have a product-led, cloud-first growth motion. The signup flow will be a fast self-service experience that lets users pick the right deployment option for them. After signup, users will discover the value of Sourcegraph in a few minutes and have a guided path to becoming a power user, that will lead them to share Sourcegraph with their teammates and power viral growth within a given customer. Sourcegraph's experience will be optimised for collaboration. This will allow Sourcegraph the company to have a product-led-growth motion and to deliver its vision to make it so everyone can code.
```
Comment (billCaplan) [inline content/strategy-goals/strategy/growth-team/index.md:5]: Might be worth a mention of new business focus vs existing customer expansion.  Not sure if that is implied by growth.

Comment (malomarrec) [reply content/strategy-goals/strategy/growth-team/index.md:5]: We are indeed focused on net new accounts, but I don't think virality can be driving that.
Given we are single-tenant, virality/feedback loop will primarily be within an account. This is required, though, to support the net new account expansion:
1. growth marketing creates signups
2. virality makes so new signup instances have > 1 user
But virality/feedback loops aren't driving net new accounts on their own

File: content/strategy-goals/strategy/growth-team/index.md [hunk 4]
```diff
@@ -39,16 +51,20 @@ Our overall strategy is to unlock the four key themes that will enable product-l
...
-
-- **Iterate on the signup experience.** Build a fast signup experience that feels like a product.
-- **Improve admin onboarding and instance configuration.**
-- **Refine user activation metrics and iterate on user onboarding and activation.**
-- **Explore feedback loops and collaboration.** The scope of this is TBD. The first step is for product/design to research the key feedback loops/collaboration moments in the current Sourcegraph experience, and to make sure they result in acquisition/activation.
+Now that we have launched an initial, semi-automated signup flow, our initial priorities are:
 
-Here's a (private) [roadmap proposal](https://docs.google.com/document/d/1HSSHT_qW3f6unEl3VXZNZi9Uw13wX-l9e0LYJEKOrdY/edit#) that will be merged in here once reviewed.
+- **Improve initial user onboarding**, in particular (a) being able to discover the value of Sourcegraph and (b) adding code. To do so, we're working on an onboarding widget ([#43596](https://github.com/sourcegraph/sourcegraph/issues/43596)), and investigating how to make adding code one-click (not started yet).
```
Comment (unknwon) [inline content/strategy-goals/strategy/growth-team/index.md]: nit: should "(a)" and "(b)" be swapped? To me, current (a) depends on (b).

File: content/strategy-goals/strategy/growth-team/index.md [hunk 5]
```diff
@@ -39,16 +51,20 @@ Our overall strategy is to unlock the four key themes that will enable product-l
...
+- **Build the infrastructure required to improve the signup and onboarding experience** ([RFC 763](https://docs.google.com/document/d/1MoDYlWBWALGDj-DJdlCvAAFpkrW0nwJL_nb0ih4uUJ0/edit#)). We will start building a cloud console API that will be be used to request and customized (eg. set subdomain), pre-configure (eg. pre-configure Google workspace SSO, pre-load OSS code to instances), and manage trial instances. This infrastructure is not growth-specific and has broad utility for Cloud instances in general.
+- **Iterate on the signup experience.** Our endgame for signup is the following:
+  - I sign in via Google Workspace or GitHub to start a trial on signup.sourcegraph.com
+  - If Google Workspace, my instance is preconfigured to allow other users to sign in via Google Workspace
+  - If GitHub, my instance is preconfigured to allow other users to sign in via GitHub
```
Comment (unknwon) [inline content/strategy-goals/strategy/growth-team/index.md]: Is this the right place to call out?

1. Only allow others with the same email domain to automatically sign in via Google Workspace
2. Only allow others within the same GitHub organization to automatically sign in via GitHub

**Review by unknwon — APPROVED** [review]

File: content/strategy-goals/strategy/growth-team/index.md (hunk 5 above)
Comment (malomarrec) [reply content/strategy-goals/strategy/growth-team/index.md]: Yes, thank you. Will clarify.

**Review by taylorsperry — APPROVED** [review]

File: content/strategy-goals/strategy/growth-team/index.md (hunk 4 above)
Comment (malomarrec) [reply content/strategy-goals/strategy/growth-team/index.md]: There's no dependency eg. if we add OSS code by default, _but_ the swap makes logical sense. Done

**Pull request merged**